torch/nn/intrinsic/__init__.py

```diff
@@ -12,6 +12,7 @@
 from torch.ao.nn.intrinsic import BNReLU3d
 from torch.ao.nn.intrinsic import LinearBn1d
 from torch.ao.nn.intrinsic import LinearLeakyReLU
+from torch.ao.nn.intrinsic import LinearTanh
```

Comment:
please revert these, we don't want to add new things to torch/nn/intrinsic folder, this is a folder that we plan to deprecate in the future

Reply:
Ok they are reverted.